### PR TITLE
Lawrence/ranking slider

### PR DESCRIPTION
**Pull request by lawrencechen14**
- Creating slider input using `range_field_tag` to replace radio buttons
- Fixed cucumber tests to account for slider

### CODE REVIEW

**Review by rejoycesong — APPROVED** [review]
Nice job on this. It'd be nice if the sliders were automatically set on the value they currently were, though.

**Review by Turese — APPROVED** [review]
Everything works fine! I'm a little concerned about the still-commented out cucumber tests, but it looks like they're for things that are already covered by other tests. Could do with some refactoring to improve code coverage later for different things, but it's okay now. I still think we might need more explanation about 1 being the best ranking but if nobody else thinks it's unclear then it's probably not a problem.

File: app/views/rankings/edit.html.erb [hunk 1]
```diff
@@ -13,6 +13,21 @@
   </div>
 <% end %>
 
+<script>
+function parseRank(rank) {
+    rank = parseInt(rank);
+    if (rank === 0) {
+        return "Can't go";
+    } else if (rank === 1) {
```
Comment (kev-chien) [inline app/views/rankings/edit.html.erb]: could have done (4 - rank). but this is fine

File: app/controllers/rankings_controller.rb [hunk 2]
```diff
@@ -115,6 +115,18 @@ def create
     redirect_to end_project_participant_ranking_path
   end
 
+  def parse_rank(rank)
+    if rank.eql? 0
+      0
+    elsif rank.eql? 1
```
Comment (kev-chien) [inline app/controllers/rankings_controller.rb]: could have done (4 - rank). but this is fine

File: app/views/rankings/edit.html.erb (hunk 1 above)
Comment (kev-chien) [inline app/views/rankings/edit.html.erb]: could have done (4 - rank). but this is fine

**Review by kev-chien — APPROVED** [review]
Looks good! Thanks Lawrence, suggested change here is just style change.

**Pull request merged**